### ISSUE TITLE
Videos stutter when auto played after opening

### ISSUE THREAD

**Issue by foss-** [stoically/humanetube#10]
This does not happen when opening video on youtube.

Firefox 102.0.1 on macOS 12.4 HumaneTube 0.1.4.

**Comment by stoically** [comment]
Yeah, the Add-on is hopelessly outdated and my custom buffer code is.. not good. Had to stop using it myself because no energy to improve it or keep it updated. I guess the better alternative is something like FreeTube or NewPipe.

**Comment by foss-** [comment]
Sure, no worries and thanks for offering this add-on in the first place.

**Comment by stoically** [comment]
Thanks, I'll keep the issue open. Maybe someone wants to pick it up.
